pkgs/development/php-packages/scrypt/default.nix

```
@@ -0,0 +1,26 @@
+{ buildPecl, lib, fetchFromGitHub }:
+
+let
+  version = "v2.0.1";
+in buildPecl {
+  inherit version;
+
+  pname = "scrypt";
+
+  src = fetchFromGitHub {
+    owner = "DomBlack";
+    repo = "php-scrypt";
+    rev = version;
```

```suggestion
  version = "2.0.1";
in buildPecl {
  inherit version;

  pname = "scrypt";

  src = fetchFromGitHub {
    owner = "DomBlack";
    repo = "php-scrypt";
    rev = "v${version}";
```

The version strring shouldn't contain the letter `v` :slightly_smiling_face:

---

thanks for the suggestion, changed it